apps/web/vercel.json

```diff
@@ -4,5 +4,13 @@
       "path": "/api/cron/calendar-cache-cleanup",
       "schedule": "0 5 * * *"
     }
-  ]
+  ],
+  "functions": {
+    "pages/api/trpc/public/[trpc].ts": {
+      "memory": 768
+    },
+    "pages/api/trpc/slots/[trpc].ts": {
+      "memory": 768
```

Comment:
We currently don't use this route from the UI but preparing this because we have a TODO to switch it from `/api/trpc/public/slots.getSchedule` to `/api/trpc/slots/getSchedule`.